benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 3.18)
+
+project(libcudacxx-benchmarks LANGUAGES CXX CUDA)
+
+set(CMAKE_BUILD_TYPE "RelWithDebInfo")
+
+set(GPU_ARCHS "70;72;75;80" CACHE STRING "List of GPU architectures to compile for.")
+
+message(STATUS "Enabled CUDA architectures:${GPU_ARCHS}")
+
+find_package(Threads REQUIRED)
+find_package(OpenMP)
+
+function(ConfigureHostBench BENCH_NAME BENCH_SRC)
```

Comment:
Would it make sense to define these functions instead in some `CMake/` utility file for reuse?

If we're going to grow samples and benchmarks and begin to have some sort of build hierarchy it might make sense to start a build utility folder.

Just a thought, not urgent or blocking.

Reply:
At this point I'd envision that all benchmark executables will be defined in this cmake file. There may be some overlap between these functions and ones used in the `samples/`, but I suspect that the functions used for defining benchmarks will be sufficiently different from samples to make it not worth trying to reuse.